tests/unit/helpers/format-message-(html)-test.js

```diff
@@ -52,24 +61,41 @@ module('format-message (html)', function (hooks) {
   });
 
   test('should handle dynamic attributes', function (assert) {
-    assert.expect(1);
+    assert.expect(2);
 
-    let output = this.intl.formatMessage(`'<'a href="{link}"'>'text'</a>'`, {
-      htmlSafe: true,
-      link: 'http://formatjs.io',
-    }).string;
+    assert.equal(
+      this.intl.formatMessage(`'<'a href="{link}"'>'text'</a>'`, {
+        htmlSafe: true,
+        link: 'http://formatjs.io',
+      }).string,
+      '<a href="http://formatjs.io">text</a>'
+    );
 
-    assert.equal(output, '<a href="http://formatjs.io">text</a>');
+    assert.equal(
+      this.intl.formatMessage(`<a href="{link}"'>text</a>`, {
```

Comment:
Prompted by #1443 I wanted to give this another read and just noticed this now.
I think the `'` is incorrect and might be masquerading a bug.

```suggestion
      this.intl.formatMessage(`<a href="{link}">text</a>`, {
```

Reply:
Ya looks like it shouldn't be there.  Without it, the test is failing?

Reply:
Unfortunately (?) the test does not fail. :/

I started #1444.